R/timeSinceFire.R

```diff
@@ -101,16 +98,19 @@ timeSinceFire <- function(input, output, session, rasters) {
 
   callModule(tilesUpdater, "rasterUpdater", proxy, urlTemplate, "Time since fire", addTilesParameters, addLayersControlParameters)
 
-  output$timeSinceFire2Hist <- renderPlot({
-    ras1 <- rasterInput()$r
-    Nbreaks <- ceiling(maxValue(ras1)/10)
-    timeSinceFireHist <- hist(ras1[], plot = FALSE, breaks = Nbreaks)
-    barplot(timeSinceFireHist$counts*prod(rasterResolution)/1e4, xlab = "Time since fire \n(Years)",
-            col = timeSinceFirePalette(1:(maxAge/10)), width = 1, space = 0, ylab = "Area (ha)")
-    axis(1, at = timeSinceFireHist$breaks/10, labels = 0:Nbreaks*10)
+  raster <- reactive(rasterInput()$r)
+
+  breaks <- reactive(ceiling(maxValue(raster())/10))
 
+  addAxisParams <- reactive({
+    numberOfBreaks <- ceiling(maxValue(raster())/10)
```

Comment:
DRI - `ceiling(maxValue(raster())/10)` is repeated above. Please reuse breaks() reactive value.

Reply:
I didn't reuse breaks(), because in general breaks might be also a vector (not a number)

Reply:
Done